### PR TITLE
US03: Signup

### PR DESCRIPTION
**Pull request by khanh000huynh**

### CODE REVIEW

File: lib/src/screens/login_screen.dart [hunk 1]
```diff
@@ -1,46 +1,14 @@
 import 'package:flutter/material.dart';
-import 'package:mobile_app/src/constants/constant_colors.dart';
```
Comment (giahy1) [inline lib/src/screens/login_screen.dart:2]: relative path

File: lib/src/widgets/custom_button.dart [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'package:mobile_app/src/constants/constant_colors.dart';
```
Comment (giahy1) [inline lib/src/widgets/custom_button.dart]: path

Comment (nntanhung) [inline lib/src/widgets/custom_button.dart]: import with relative path

File: lib/src/app.dart [hunk 3]
```diff
@@ -1,25 +1,39 @@
...
-import '../src/screens/signup_screen.dart';
+import '../src/blocs/signup_bloc/signup_bloc.dart';
+import '../src/screens/home_screen.dart';
+import '../src/services/user_services.dart';
+import 'screens/login_screen.dart';
+import 'screens/onboarding_screen.dart';
+import 'screens/signup_screen.dart';
 
 class App extends StatelessWidget {
+  final UserServices userServices = UserServices();
```
Comment (thanhphamtrung) [inline lib/src/app.dart]: final userServices = UserServices(); 
=> shorter but look better

File: lib/src/blocs/signup_bloc/signup_event.dart [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import 'package:equatable/equatable.dart';
+
+import '../../models/user_model.dart';
+
+abstract class SignupEvent extends Equatable {
+  final UserModel userModel;
+
+  const SignupEvent({this.userModel = const UserModel()});
```
Comment (thanhphamtrung) [inline lib/src/blocs/signup_bloc/signup_event.dart]: should we move this to SignupRequested?

File: lib/src/services/user_services.dart [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:mobile_app/src/models/user_model.dart';
```
Comment (thanhphamtrung) [inline lib/src/services/user_services.dart]: .

File: lib/src/app.dart (hunk 3 above)
Comment (khanh000huynh) [reply lib/src/app.dart]: I removed it.

File: lib/src/services/user_services.dart (hunk 5 above)
Comment (khanh000huynh) [reply lib/src/services/user_services.dart]: My bad.

**Pull request merged**